### ISSUE TITLE
VW Objective with train and holdout set

### ISSUE THREAD

**Issue by vsuthichai** [eHarmony/spotz#4]
Implement a VW objective that accepts a training set and test set on which to train and evalutate a VW model.
